SConstruct

```diff
@@ -270,9 +270,11 @@ def scons():
 
     set_defaults(env)
 
+    build_prefix = prereqs.get_src_build_dir()
```

Comment:
(pylint-no-member) Instance of 'PreReqComponent' has no 'get_src_build_dir' member